nextcord/application_command.py

```diff
@@ -111,6 +117,46 @@ def _cog_special_method(func: FuncT) -> FuncT:
     return func
 
 
+class InteractionDefault:
+    # TODO: fix docstring lol
+    """A class that makes all thing default.
+
+    To make a default from the :class:`Interaction`, you can do like this:
+    .. code-block:: python3
+        class InteractionUser(InteractionDefault):
+            def default(self, interaction: Interaction):
+                return interaction.user
+
+        @bot.slash_command()
+        async def balance(interaction, user = SlashOption(name="user", description="The user to check the balance", default=InteractionUser):
+            ...
+    """
+
+    def default(self, interaction: Interaction) -> None:
+        raise NotImplementedError
+
+
+class InteractionAuthor(InteractionDefault):
+    """Return the author of the interaction."""
```

Comment:
```suggestion
    """Return the user of the interaction."""
```

Reply:
This was never actually resolved with any communication or commits @MaskDuck.